src/containers/Webmap/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {FC, useEffect, useState} from 'react';
+import axios from 'axios';
+import {Icon} from 'leaflet';
+import {Map, Marker, Popup, TileLayer} from 'react-leaflet';
+import './Webmap.scss';
+import * as apiEndpoints from '../../constants/api-endpoints';
+
+export const bankIcon = new Icon({
+  iconSize: [20, 20],
+  iconUrl: '/bank1.svg',
+});
+
+export const validatorIcon = new Icon({
+  iconSize: [20, 20],
+  iconUrl: '/validator.svg',
+});
+
+const Webmap: FC = () => {
+  interface APIResponse {
+    account_number: number;
+    city: string;
+    country: string;
+    default_transaction_fee: number;
+    lat: number;
+    lon: number;
+    org: string;
+    query: string;
+    root_account_file: string;
+  }
+
+  const [state, setState] = useState<APIResponse[]>([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      const nodeResponse = await axios.all([
+        axios.get(apiEndpoints.VALIDATOR_ENDPOINT),
+        axios.get(apiEndpoints.BANK_ENDPOINT),
+      ]);
+      const nodeResponses = nodeResponse[0].data.results.concat(nodeResponse[1].data.results);
+      const validatorIps = [];
+      for (let i = 0; i < nodeResponses.length; i += 1) {
+        const ipAddress = nodeResponses[i].ip_address;
+        validatorIps.push({query: ipAddress});
+      }
+      const resultBatch = await axios.post(apiEndpoints.IPAPI_ENDPOINT, validatorIps);
+      const merged = nodeResponses.map((result: any, i: string | number) => ({
+        ...result,
+        ...resultBatch.data[i],
+      }));
+      setState(merged);
+    }
+    fetchData();
+  }, []);
+
+  return (
+    <Map center={[37, -30]} zoom={3}>
+      <TileLayer
+        attribution="Tiles &copy; Esri &mdash; Esri, DeLorme, NAVTEQ"
+        url="https://server.arcgisonline.com/ArcGIS/rest/services/Canvas/World_Light_Gray_Base/MapServer/tile/{z}/{y}/{x}'"
+      />
+      {state.map((node) => (
+        <Marker
+          key={node.query}
+          position={[node.lat + Math.random() * (0.2 - 0.1) + 0.1, node.lon]}
+          icon={node.root_account_file != null ? validatorIcon : bankIcon}
```

Comment:
```suggestion
          icon={node.root_account_file === null ? bankIcon : validatorIcon}
```

Reply:
I tried this suggestion but it wasn't working as expected. I think I'll leave it as is for now, unless you'd like me to investigate further.